src/Post/Post.js

```diff
@@ -1,7 +1,5 @@
-import { ClassNames } from '@emotion/react';
 import MoreHoriz from '@mui/icons-material/MoreHoriz';
-import classNames from 'classnames';
-import classnames from 'classnames';
+import { default as classNames, default as classnames } from 'classnames';
```

Comment:
This is because you're using them once as classNames and once as classnames. One import should be enough.